src/main/java/org/gridsuite/shortcircuit/server/entities/AnalysisParametersEntity.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.shortcircuit.server.entities;
+
+import com.powsybl.shortcircuit.InitialVoltageProfileMode;
+import com.powsybl.shortcircuit.StudyType;
+import jakarta.persistence.*;
+import lombok.*;
+import lombok.experimental.Accessors;
+import org.gridsuite.shortcircuit.server.dto.ShortCircuitPredefinedConfiguration;
+
+import java.util.UUID;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
```

Comment:
you ?

Reply:
see https://github.com/gridsuite/shortcircuit-server/pull/81#discussion_r1629473725